src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html

```diff
@@ -1,39 +1,55 @@
+            <input class="col-md-2 details-checkbox" type="checkbox" ng-model="person.isWeb">
+        </div>
+        <div class="form-group">
+            <label class="control-label col-md-6">Tester:</label>
+            <input class="col-md-2 details-checkbox" type="checkbox" ng-model="person.isTest">
+        </div>
+        <hr>
+        <div class="form-group">
+            <label class="col-md-4 control-label">Band level:</label>
+            <div class="col-md-8">
+                <div class="ui-select">
+                    <select class="form-control" ng-model="person.bandLevel" ng-options="key as value for (key, value) in BandLevelEnum" ng-init="person.bandLevel = '6';">
+                    </select>
+                </div>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-md-4 control-label">E-mail:</label>
+            <div class="col-md-8">
+                <input class="form-control input" type="email" ng-model="person.email" required/>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-md-4 control-label">First name:</label>
+            <div class="col-md-8">
+                <input class="form-control input" type="text" ng-model="person.firstName" required/>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="col-md-4 control-label">Last name:</label>
+            <div class="col-md-8">
+                <input class="form-control input" type="text" ng-model="person.lastName" required/>
+            </div>
+        </div>
+        <div class="center btn-wrapper">
+            <input class="btn btn-primary" type="submit" ng-click="create(person)" value="Submit"
+               ng-disabled="form.$invalid || createInterviewerPromise.resolved"/>
+            <input class="btn btn-default" type="button" value="Cancel" ng-click="goHome()">
+            <img ng-if="createInterviewerPromise.resolved"
+                 src="data:image/gif;base64,R0lGODlhEAAQAPIAAP///wAAAMLCwkJCQgAAAGJiYoKCgpKSkiH/C05FVFNDQVBFMi4wAwEAAAAh/hpDcmVhdGVkIHdpdGggYWpheGxvYWQuaW5mbwAh+QQJCgAAACwAAAAAEAAQAAADMwi63P4wyklrE2MIOggZnAdOmGYJRbExwroUmcG2LmDEwnHQLVsYOd2mBzkYDAdKa+dIAAAh+QQJCgAAACwAAAAAEAAQAAADNAi63P5OjCEgG4QMu7DmikRxQlFUYDEZIGBMRVsaqHwctXXf7WEYB4Ag1xjihkMZsiUkKhIAIfkECQoAAAAsAAAAABAAEAAAAzYIujIjK8pByJDMlFYvBoVjHA70GU7xSUJhmKtwHPAKzLO9HMaoKwJZ7Rf8AYPDDzKpZBqfvwQAIfkECQoAAAAsAAAAABAAEAAAAzMIumIlK8oyhpHsnFZfhYumCYUhDAQxRIdhHBGqRoKw0R8DYlJd8z0fMDgsGo/IpHI5TAAAIfkECQoAAAAsAAAAABAAEAAAAzIIunInK0rnZBTwGPNMgQwmdsNgXGJUlIWEuR5oWUIpz8pAEAMe6TwfwyYsGo/IpFKSAAAh+QQJCgAAACwAAAAAEAAQAAADMwi6IMKQORfjdOe82p4wGccc4CEuQradylesojEMBgsUc2G7sDX3lQGBMLAJibufbSlKAAAh+QQJCgAAACwAAAAAEAAQAAADMgi63P7wCRHZnFVdmgHu2nFwlWCI3WGc3TSWhUFGxTAUkGCbtgENBMJAEJsxgMLWzpEAACH5BAkKAAAALAAAAAAQABAAAAMyCLrc/jDKSatlQtScKdceCAjDII7HcQ4EMTCpyrCuUBjCYRgHVtqlAiB1YhiCnlsRkAAAOwAAAAAAAAAAAA=="/>
+        </div>
+    </form>
+</div>
```

Comment:
Isn't it copy pasted from another partial?
Can't we use the existing html partial instead of creating identical one?